source/developer-guides/debug.rst

```diff
@@ -0,0 +1,197 @@
+.. _debug:
+
+Source Level Debugging
+--------------
+
+This explains how to perform basic debugging operation with Intel UDK Debugger on SBL over serial port. (used Apollo Lake (ABL) as example)
```

Comment:
* Use substitution token for 'SBL': |SPN|. There are many examples in the source.
* Remove (...) as the content is fairly generic overall. We could add more concrete steps for UP2 later.

Reply:
fixed